readme.md

```diff
@@ -44,7 +44,7 @@ The returned object specifies a level of support for color through a `.level` pr
 
 It obeys the `--color` and `--no-color` CLI flags.
 
-For situations where using `--color` is not possible, add an environment variable `FORCE_COLOR` with any value to force color. Trumps `--no-color`.
+For situations where using `--color` is not possible, add an environment variable `FORCE_COLOR` with any value to force color. If `FORCE_COLOR` is set to `0`, colors are force disabled. Trumps `--no-color` / color flags, respectively.
```

Comment:
I think we should drop `with any value`, and instead say `FORCE_COLOR=1` to force color, `FORCE_COLOR=0` to force disable colors, and don't use it for auto-detection (for clarity).